.sanity-ansible-ignore-2.12.txt

```diff
@@ -0,0 +1,65 @@
+plugins/module_utils/network_lsr/nm/__init__.py empty-init!skip
+plugins/module_utils/network_lsr/nm/__init__.py import-2.6!skip
+plugins/module_utils/network_lsr/nm/__init__.py import-2.7!skip
+plugins/module_utils/network_lsr/nm/__init__.py import-3.5!skip
```

Comment:
Why are there skips for Python 3.5?

Reply:
Because otherwise ansible-test complains about it

Reply:
for example:
```
Running sanity test "import" on Python 3.5
ERROR: Found 6 import issue(s) on python 3.5 which need to be resolved:
ERROR: plugins/module_utils/network_lsr/nm/__init__.py:7:0: traceback: ImportError: No module named 'gi' (at plugins/module_utils/network_lsr/nm/client.py:13:0)
ERROR: plugins/module_utils/network_lsr/nm/active_connection.py:13:0: traceback: ImportError: No module named 'gi' (at plugins/module_utils/network_lsr/nm/client.py:13:0)
...
```

Reply:
Is it possible to tell ansible-lint the target python versions instead?

Reply:
> Is it possible to tell ansible-lint the target python versions instead?

The problem is that we don't want to use the ansible-test import/compile checking, since we already have unit tests for that.  It would be quite a bit of work to make ansible-test import and compile checks pass, and it isn't really necessary.

Reply:
Note that ansible-test is really designed for checking *collections*, and primarily for checking/testing collections that provide *plugins* which are used directly by end users.  System roles does not provide plugins that are intended to be used directly by end users - our modules are intended to be used only within the context of the enclosing role, and the role ensures that the proper dependencies are in place in order to run the python modules, and we test that in unit tests and/or integration tests.  Therefore, supporting ansible-test import and compile checks are more trouble than they're worth, so we skip them.